pkg/apis/resource/validation/validation_resourceclaim_test.go

```diff
@@ -308,7 +308,7 @@ func TestValidateClaimUpdate(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			scenario.oldClaim.ResourceVersion = "1"
 			errs := ValidateClaimUpdate(scenario.update(scenario.oldClaim.DeepCopy()), scenario.oldClaim)
-			assert.Equal(t, scenario.wantFailures, errs)
+			assert.Equal(t, len(scenario.wantFailures), len(errs))
```

Comment:
That's a fairly weaker assertino, can we go through the list of failures and check for `strings.Contains` since I'm assuming that's the problem you're trying to solve?

Reply:
And throughout the tests